src/mixins/raster-layer-point-mixin.js

```diff
@@ -561,7 +561,7 @@ export default function rasterLayerPointMixin(_layer) {
       _layer
         .yDim()
         .groupAll()
-        .valueAsync()
+        .valueAsync(false, false, false, layerName)
```

Comment:
The rest of these mods just hand in the layerName to functions that need 'em. This was the only slightly noteworthy change, in that valueAsync had to be modded to take a layerName after all the defaults.